### ISSUE TITLE
Cyan output of version number does not allow using it in the command line

### ISSUE THREAD

**Issue by fervic** [jpettersson/autoversion#10]
Hi there,

Is there any special reason why the version number is output in cyan? I was trying to do the following and didn't work:

```
git checkout -b release-`autoversion`
fatal: git checkout: we do not like 'release-0.1.0' as a branch name.
```

And I think it is because of the color.

**Comment by jpettersson** [comment]
@fervic Only aesthetic reasons. If it breaks usage it should be possible to remove it. I'll look into ways of optionally stripping the color. Maybe a flag.

**Comment by fervic** [comment]
No, colors are good. I was thinking more on passing over a --no-color
option in cases you want to programmatically use the output. I wanted to
work on this myself, just haven't taken out the time for it.

**Comment by jpettersson** [comment]
@fervic I realized I was a bit unclear and edited my message after sending it. `--no-color` sounds like a good idea.
